deepmd/tf/descriptor/se_atten.py

```diff
@@ -558,17 +563,18 @@ def build(
             if nvnmd_cfg.restore_descriptor:
                 davg, dstd = build_davg_dstd()
             check_switch_range(davg, dstd)
-        with tf.variable_scope("descrpt_attr" + suffix, reuse=reuse):
-            if davg is None:
-                davg = np.zeros([self.ntypes, self.ndescrpt])  # pylint: disable=no-explicit-dtype
-            if dstd is None:
-                dstd = np.ones([self.ntypes, self.ndescrpt])  # pylint: disable=no-explicit-dtype
+        with tf.variable_scope("descrpt_attr", reuse=reuse):
+            t_ntypes = tf.constant(self.ntypes, name="ntypes", dtype=tf.int32)
             t_rcut = tf.constant(
                 np.max([self.rcut_r, self.rcut_a]),
                 name="rcut",
                 dtype=GLOBAL_TF_FLOAT_PRECISION,
             )
-            t_ntypes = tf.constant(self.ntypes, name="ntypes", dtype=tf.int32)
+        with tf.variable_scope("descrpt_attr" + suffix, reuse=reuse):
```

Comment:
I am not sure the reason for changing the scope name, but I believe this is not what you expect: when a suffix is empty, the second variable_scope will create `descrpt_attr_1/xxx` tensor instead of `descrpt_attr/xxx`, which I believe is what you expect. See TensorFlow documentation: https://www.tensorflow.org/api_docs/python/tf/compat/v1/variable_scope
We had a similar issue in #3928, fixed in #3930.